src/compas_fab/backends/ros/messages/__init__.py

```diff
@@ -8,7 +9,6 @@
 from .octomap_msgs import *
 from .sensor_msgs import *
 from .shape_msgs import *
-from .std_msgs import *
```

Comment:
why is this? imports are sorted alphabetically, so, next time we run auto-sort it will go back

Reply:
I'll let @romanarust  handle this

Reply:
👍 Done.